src/homepage/ForDataUsers/DataBrowser.jsx

```diff
@@ -9,7 +9,7 @@ export const DataBrowser = () => (
   <ExpandableCard
     id="home-expand-data-browser"
     title="HMDA Data Browser"
-    description="A suite of tools that allows financial institutions to filter, summarize, download, and visualize HMDA datasets."
+    description="A suite of tools that allows users to filter, summarize, download, and visualize HMDA datasets."
```

Comment:
Was there further discussion to update this text? The text to be changed was an update to the original via feedback from the powerpoint.

Reply:
This was an update from @kgudel in https://github.com/cfpb/hmda-frontend/issues/1510.  Makes it clear that the tools aren't exclusively for FIs.

Reply:
I understand, thanks for the update. Looks good!